tests/test_sbi.py

```diff
@@ -503,12 +693,12 @@ def test_yaml():
         thetafid_file='thetaobs.npy',
```

Comment:
Shouldn't thetaobs.npy be saved before this? So the test is self-consistent.

Reply:
Although there was no attempt to access this file until later in the test function, after which this file had been created, I agree that it would be nicer for the file to have been made before this point. I have therefore moved its creation to before this point.